AnaProd/MergeNtuples.py

```diff
@@ -122,4 +129,23 @@ def merge_ntuples(df):
     if args.useUproot:
         ConvertUproot.toUproot(tmpFileName, args.outFile)
         #if os.path.exists(args.outFile):
-        #    os.remove(tmpFileName)
\ No newline at end of file
+        #    os.remove(tmpFileName)
+
+    if args.outFiles != None and len(args.outFiles) > 0:
+        for obj_name, obj_desc in objects.items():
+            print(f"At splitting for {obj_name} {obj_desc}")
+            # We want to make this general to also split MC later, but these object names could cause an issue?
```

Comment:
If there is more than one TTree in the file -> raise an error, since the current splitting implementation will not work.